app/models/ems_refresh.rb

```diff
@@ -37,6 +37,9 @@ def self.queue_refresh(target, id = nil, opts = {})
       h[e] << t unless e.nil?
```

Comment:
sorry...
would it make sense to do this here?

```suggestion
      h[e] << t if e&.supports?(:refresh_ems)
```

Reply:
Yeah good call, updated
NOTE I left the `nil` check and the `supports` check as separate since I thought it was more obvious than implicitly checking for `nil&.supports?`